# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/mine/mine-md5-cdefs-legacy.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- a/md5.c	2014-04-28 16:22:50.653150935 +0200
+++ b/md5.c	2014-04-28 16:23:50.329249501 +0200
@@ -39,6 +39,11 @@
 # define SWAP(n) (n)
 #endif
 
+/* glibc legacy defined in <sys/cdefs.h> but obsolete */
+#ifndef __P
+# define __P(args) args
+#endif
+
 typedef unsigned int md5_uint32;
 
 /* Structure to save state of computation between the single steps.  */
